tcms/kiwi_auth/admin.py

```diff
@@ -144,6 +144,19 @@ def delete_model(self, request, obj):
         delete_user(obj)
 
 
+class GroupAdmin(admin.ModelAdmin):
+
+    def get_readonly_fields(self, request, obj=None):
+        readonly_fields = super().get_readonly_fields(request, obj)
+
+        if hasattr(obj, 'name') and obj.name in ['Tester', 'Administrator']:
+            readonly_fields = readonly_fields + ('name',)
+
+        return readonly_fields
```

Comment:
Name field moves to the bottom which is very unnatural, should stay on the top and for that you probably need fieldsets defined too.